models/general-models.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require("../endpoints.json");
+
+exports.selectAllEndpoints = () => {
```

Comment:
as we aren't interacting with the database for this endpoint, we don't actually need a model here - we can directly send this from the controller! Free to remove if you'd like to